.github/workflows/pypi-publish.yaml

```diff
@@ -1,5 +1,14 @@
 on:
   workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Deployment environment
+        options:
+          - test
+          - prod
+          - skip
+        default: test
```

Comment:
In testing the workflow, I realized that it's annoying to have to test deployments via the `release` and `prerelease` events exclusively. This workflow input gives us the ability to push to PyPI without needing to cut a GitHub release. We should avoid doing this when possible, for the sake of consistency, so it's not documented -- but I think having the functionality will be useful in cases where we make a mistake during the release process and need to fix it manually.